components/TranscriptControl.tsx

```diff
@@ -7,12 +7,19 @@ type TranscriptControlParams = {
 
 export default function TranscriptControl({onTimeStampSelected, children} : TranscriptControlParams) {
   function handleClick(e): void {
-    if (e.target.tagName !== 'SPAN') {
+    // Search for nearest span.
```

Comment:
![tumblr_lkqm0enAZt1qh59n0o1_500](https://github.com/SPS-By-The-Numbers/transcripts/assets/1922786/646e19b7-a7b3-4681-b81b-30d2fb816511)